### ISSUE TITLE
有个奇妙的发现……QKeyEvent貌似要在widget使用了layout的情况下才会对键盘有反应. 另外在image上画完了之后记得要重新setPixmap才能显示出来。

### ISSUE THREAD

**Issue by tzhangam** [tzhangam/Project_2#11]

**Comment by thomaszhouan** [comment]
For QKeyEvent, the keyboard focus should be set before the widget can receive keyboard event.
